twoliter/embedded/imghelper

```
@@ -169,6 +169,42 @@ symlink_image() {
   done
 }
 
+install_ca_certs() {
+  local root_mount
+  root_mount="${1:?}"
+
+  local ca_certs_default_bundle="/usr/share/bottlerocket/ca-certificates/ca-bundle.crt"
+  local ca_certs_default_licenses="/usr/share/licenses/ca-certificates"
+
+  local ca_certs_bundle="${HOME}/certs/ca-bundle.crt"
+  local ca_certs_installdir="/usr/share/factory/etc/pki/tls/certs"
+
+  install -d "${root_mount}${ca_certs_installdir}"
+  if [[ -f "${ca_certs_bundle}" ]]; then
+    install -p -m 0644 \
+      "${ca_certs_bundle}" \
+      "${root_mount}${ca_certs_installdir}"
+  elif [[ -f "${ca_certs_default_bundle}" ]]; then
+    install -p -m 0644 \
+      "${ca_certs_default_bundle}" \
+      "${root_mount}${ca_certs_installdir}"
+    # Make sure we're installing the licenses too.
+    if [[ ! -s "${ca_certs_default_licenses}" ]]; then
+      echo "no CA certificates attribution found" >&2
+      exit 1
+    fi
+    install -p -m 0644 \
+      -D "${ca_certs_default_licenses}"/* \
+      -t "${root_mount}/usr/share/licenses/ca-certificates"
+  else
+    echo "no CA certificates found" >&2
+    exit 1
+  fi
+
+  openssl verify \
+    "${root_mount}${ca_certs_installdir}/ca-bundle.crt" >/dev/null || exit 1
```

can just let the failure propagate back to the caller:
```suggestion
  openssl verify \
    "${root_mount}${ca_certs_installdir}/ca-bundle.crt" >/dev/null
```